### DISCUSSION TITLE
오타 제보 (Typo report)

### DISCUSSION THREAD

**Discussion by 33577** [prosyslab-classroom/cs524-program-analysis#443] [General]
안녕하세요, 
몇가지 가벼운 오타를 찾아 제보드립니다. 
1. "Wideining"
  - 올바른 표현: "Widening"
  - 위치: README > Schedule > 7th rows
3. "12."
  - 올바른 표현: "13."
  - 위치: 13 슬라이드 모든 페이지 하단 꼬리글의 단원 표시 

감사합니다 : )
(eng) 

Hi :) 
This is a report for simple typos. 

1. "Wideining"(should be "Widening") in the README > Schedule > 7th rows
2. "12."(should be "13.")  in the footer of 13 slides

Thanks : )

**Comment by KihongHeo** [comment]
고맙습니다.
